core/sql/executor/ExExeUtilLoad.cpp

```diff
@@ -3459,6 +3458,25 @@ short ExExeUtilLobUpdateTcb::work()
             char outLobHandle[LOB_HANDLE_LEN];
             Int32 outHandleLen;
             Int64 requestTag = 0;
+            if (lobTdb().lobLocking())
+              {
+                ExpLOBoper::genLobLockId(uid,lobNum,lobLockId_);
+                NABoolean found = FALSE;
+                retcode = SQL_EXEC_CheckLobLock(lobLockId_ , &found);
+                if (! retcode && !found) 
+                  {    
+                    retcode = SQL_EXEC_SetLobLock(lobLockId_);
```

Comment:
Might this call fail? It looks like we ignore the retcode in this code path.

Reply:
Will add the error condition for non zero retcode too.